app/github_public_repo/estimate_csv_creator.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import pandas as pd
+from flask import current_app as app
+
+from app.database_etl.airtable_records_handler import get_all_records, airtable_get_request
+
+# Load env variables for Airtable request
+AIRTABLE_API_KEY = app.config['AIRTABLE_API_KEY']
+AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL = app.config['AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL']
+
+# Get columns that should be pulled into CSV
+headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+data = airtable_get_request(AIRTABLE_GITHUB_CSV_FIELDS_REQUEST_URL, headers)
+
+# Get csv columns and create csv
+try:
+    # Extract fields
+    records = data['records']
+    fields = [x['fields']['Field Name'] for x in records]
+    logging.info("Successfully retrieved field names from Airtable")
+
+    # Get estimates from Rapid Review: Estimates table for specified fields
+    csv_records = get_all_records(fields)
+
+    # Convert to df, reorder cols alphabetically and as save as csv
+    csv_records_df = pd.DataFrame.from_dict(csv_records)
+    csv_records_df = csv_records_df.reindex(sorted(csv_records_df.columns), axis=1)
+    csv_records_df.to_csv('GitHub CSV.csv', index=False)
```

Comment:
Will there be any issues with specifying the output file path like this (i.e. if we invoke this script from a few different python paths)

Reply:
i don't think so, it will just save the csv under this name in the same directory wherever the script is being called right. this means that the script caller just needs to keep track of where it will be saved. where will the script be called from when we run it as a github actions @annoyingpr ? we will need to run the script, generate the csv, and then know the file path of the script so we can upload the csv to the repo.

Reply:
okay just checking, will leave it up to @annoyingpr to ensure this won't be an issue

Reply:
the script will most likely be running from root, but currently trying to figure out if we can setup an absolute path for this CSV

Reply:
this is resolved. we are now using abs path.